src/modelbench/hazards.py

```diff
@@ -57,12 +58,23 @@ def __repr__(self):
 
 
 class SafeHazard(HazardDefinition, ABC):
+
+    def __init__(self):
+        # We expose these so that a BenchmarkDefinition can look into the specifics
+        # of what got into scoring calculations
+        super().__init__()
+        self.num_scored_items = 0
+        self.num_safe_items = 0
```

Comment:
Is there a reason these numbers can't go into the HazardScore? I was thinking of these objects as immutable and their methods as idempotent, with the state living in the *Score objects.